guides/periodic_jobs.md

```diff
@@ -0,0 +1,119 @@
+# Periodic Jobs
+
+Oban's [cron plugin](`Oban.Plugins.Cron`) registers workers that run on a cron-like schedule and enqueues jobs automatically.
+
+Periodic jobs are declared as a list of `{cron, worker}` or `{cron, worker, options}` tuples:
+
+```elixir
+config :my_app, Oban,
+  repo: MyApp.Repo,
+  plugins: [
+    {Oban.Plugins.Cron,
+     crontab: [
+       {"* * * * *", MyApp.MinuteWorker},
+       {"0 * * * *", MyApp.HourlyWorker, args: %{custom: "arg"}},
+       {"0 0 * * *", MyApp.DailyWorker, max_attempts: 1},
+       {"0 12 * * MON", MyApp.MondayWorker, queue: :scheduled, tags: ["mondays"]},
+       {"@daily", MyApp.AnotherDailyWorker}
+     ]}
+  ]
+```
+
+The crontab would insert jobs as follows:
+
+  * `MyApp.MinuteWorker` — Inserted once every minute
+  * `MyApp.HourlyWorker` — Inserted at the first minute of every hour with custom arguments
+  * `MyApp.DailyWorker` — Inserted at midnight every day with no retries
+  * `MyApp.MondayWorker` — Inserted at noon every Monday in the "scheduled" queue
+  * `MyApp.AnotherDailyWorker` — Inserted at midnight every day with no retries
+
+The crontab format respects all [standard rules][cron] and has one minute resolution. Jobs are
+considered unique for most of each minute, which prevents duplicate jobs with multiple nodes and
+across node restarts.
```

Comment:
Oops, this is outdated. The bit about "jobs are considered unique for most of each minute" isn't true. My comment below explains it a little more, but essentially the combination of leadership and only scheduling at the top of the minute prevents this.